v2/lib/load/config.js

```diff
@@ -25,8 +25,6 @@ const REQUIRED_FIELDS = [
 
 const OPTIONAL_FIELDS = [
   'algolia',
-  'chainWebpack',
```

Comment:
Removed this because next PR is going to allow overriding webpack config through plugins